Makefile.am

```diff
@@ -48,6 +48,7 @@ LDADD = \
 
 # keep me sorted
 bin_PROGRAMS = \
+    tools/aux/tpm2_print \
```

Comment:
Did we decide that there would be an aux sub-directory? I remember the discussion but I had that at the end we decided to just have all the tools under tools.

Reply:
I don't think we ever concluded, I'm OK with an aux tool directory, what about you @martinezjavier ?

Reply:
@williamcroberts I'm also OK with it. I just asked since it was not clear to me that there was an agreement on that.